config/webhook/manifests.yaml

```diff
@@ -51,7 +71,6 @@ webhooks:
     operations:
     - CREATE
     - UPDATE
-    - DELETE
```

Comment:
I don't think the validation webhook should validate deletes but please correct me if I'm wrong